### ISSUE TITLE
Distinctions between pbcorePart and pbcoreInstantiation

### ISSUE THREAD

**Issue by awead** [WGBH/PBCore2.0#64]
2.0 schema allows for start/stop time annotation attributes in the pbcoreInstantiation element.  This seems to duplicate the role of pbcorePart, which has the same attributes intended for marking segments.
Unless this contradicts accepted norms, can the new schema be more explicit and not allow for such time attributes in the instantiation element?  This would solidify the concept that instantiation elements are for complete media objects--whether physical or digital--and pbcorePart elements are to define segments within, and potentially across, those objects.

**Comment by AllisonAnn** [comment]
I have a question about pbcorePart.  How is it different from
pbcoreRelation=hasPart?  It seems redundant, and confusing.  Is there a
differentiation between the use of the two that isn't apparent?

**Comment by kvanmalssen** [comment]
I agree with Adam that this is the distinction between time reference in pbcoreInstantiation and pbcorePart is not clear. I would be in favor of deciding on one or the other to improve consistency. Adam's proposal to remove the start/end attributes makes sense to me. 

Allison's comment is also a valid one. Both issues point to the confusion on the use of the "part" concepts that can be found in several places in PBCore. I would add to this attributes for start/end time that are found in elements like pbcoreTitle and others. Furthermore, I think it is problematic to reference start/end times in relation to Intellectual Content/Property, as there could be multiple instantiations that don't have the exact same time reference. I think this is a separate issue to raise.

**Comment by GlennClatworthy** [comment]
I think of pbcorePart as a demarcation of content areas, for example, individual NewsHour segments. Staff in our packaging department would probably see the pbcoreInstantiation element as a way to document the technical elements of "containerized" programs, like the body vs. components like underwriting credits and promos.

**Comment by johnnypass** [comment]
To Kara's comment about start/end times:  How different are multiple Instantiations allowed to be before they become single Instantiations and related by something like pbcoreRelation=IsVersionOf?  For example, I'd consider a raw tape and a final mix of the same recording multiple Instantiations of a single asset, yet they're likely to have different start/end times AND different pbcoreContributors.

So maybe we can make a better definitions for pbcoreInstantiations and pbcoreRelations and pbcorePart? For example - 

pbcoreInstantiation
The element pbcoreInstantiation is a container for sub-elements which describe an instance of a media item, i.e., a manifestation of it in the real world. pbcoreDescriptionDocument may contain multiple pbcoreInstantiations, provided that each pbcoreInstantiation shares the same Intellectual Content/Property elements.
